src/System.Net.Http/src/System/Net/Http/Headers/KnownHeaders.cs

```diff
@@ -114,6 +114,22 @@ public StringAccessor(string s)
             public char CharAt(int index) => _string[index];
         }
 
+        // Can't use Span here as it's unsupported.
```

Comment:
If BytePtrAccessor contained a Span, then it would be a ref-like struct and could not implement an interface.